Doc/library/inspect.rst

```diff
@@ -55,6 +55,10 @@ attributes (see :ref:`import-mod-attrs` for module attributes):
 |           | __module__        | name of module in which   |
 |           |                   | this class was defined    |
 +-----------+-------------------+---------------------------+
+|           | __type_params__   | :pep:`695` type           |
```

Comment:
We shouldn't link to PEPs as long-term documentation. (Concrete reason: PEP 696 will soon change these again, and PEP 695 won't mention it.) Maybe link to https://docs.python.org/3.12/reference/compound_stmts.html#type-params ?